src/Sylius/Bundle/AssociationBundle/composer.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "sylius/association-bundle",
+    "type": "symfony-bundle",
+    "description": "Products association for your Symfony2 applications.",
+    "keywords": ["shop", "ecommerce", "products", "product", "assortment", "association"],
+    "homepage": "http://sylius.org",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Paweł Jędrzejewski",
+            "homepage": "http://pjedrzejewski.com"
+        },
+        {
+            "name": "Leszek Prabucki",
+            "email": "leszek.prabucki@gmail.com"
+        },
+        {
+            "name": "Łukasz Chruściel",
+            "email": "lukasz.chrusciel@lakion.com"
+        },
+        {
+            "name":     "Sylius project",
+            "homepage": "http://sylius.org"
+        },
+        {
+            "name":     "Community contributions",
+            "homepage": "http://github.com/Sylius/Sylius/contributors"
+        }
+    ],
+    "require": {
+        "php": "^5.5.9|^7.0",
+
+        "symfony/framework-bundle": "^2.7.7",
+        "stof/doctrine-extensions-bundle": "~1.1",
+        "symfony/form": "^2.7.7",
+        "symfony/validator": "^2.7.7",
+        "sylius/resource-bundle": "0.17"
+    },
+    "require-dev": {
+        "phpspec/phpspec": "~2.4"
+    },
+    "config": {
+        "bin-dir": "bin"
+    },
+    "autoload": {
+        "psr-4": { "Sylius\\Bundle\\ArchetypeBundle\\": "" }
+    },
+    "autoload-dev": {
+        "psr-4": { "Sylius\\Bundle\\ArchetypeBundle\\spec\\": "spec/" }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "0.17-dev"
+        }
+    },
+    "minimum-stability": "dev"
```

Comment:
This is how Symfony does it and makes it easier to install dev versions of bundles, as soon as we tag v0.17.0 it will install stable.